pkg/config/datafileprojectconfig/mappers/experiment.go

```diff
@@ -23,8 +23,9 @@ import (
 )
 
 // MapExperiments maps the raw experiments entities from the datafile to SDK Experiment entities and also returns a map of experiment key to experiment ID
-func MapExperiments(rawExperiments []datafileEntities.Experiment, experimentGroupMap map[string]string) (experimentMap map[string]entities.Experiment, experimentKeyMap map[string]string) {
+func MapExperiments(rawExperiments []datafileEntities.Experiment, experimentGroupMap map[string]string) (experiments []entities.Experiment, experimentMap map[string]entities.Experiment, experimentKeyMap map[string]string) {
```

Comment:
Does this cause any backward compatibility issues? This is public facing and is changing the return types.

Reply:
`mappers` package and everything inside it is for internal use only. We use it only to create projectConfig.